2019-09-20 13:58:37,686 log.log_response 228 WARNING => Not Found: /
2019-09-20 13:58:37,686 basehttp.log_message 154 WARNING => "GET / HTTP/1.1" 404 2035
2019-09-20 13:58:37,836 log.log_response 228 WARNING => Not Found: /favicon.ico
2019-09-20 13:58:37,836 basehttp.log_message 154 WARNING => "GET /favicon.ico HTTP/1.1" 404 2086
2019-09-20 13:58:42,209 log.log_response 228 WARNING => Not Found: /account
2019-09-20 13:58:42,210 basehttp.log_message 154 WARNING => "GET /account HTTP/1.1" 404 2074
2019-09-20 13:58:46,161 log.log_response 228 WARNING => Not Found: /account/
2019-09-20 13:58:46,162 basehttp.log_message 154 WARNING => "GET /account/ HTTP/1.1" 404 3097
2019-09-20 13:58:50,150 basehttp.log_message 154 INFO    => "GET /account/login HTTP/1.1" 301 0
2019-09-20 13:58:50,160 basehttp.log_message 154 INFO    => "GET /account/login/ HTTP/1.1" 200 2872
2019-09-20 13:58:50,174 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap.css HTTP/1.1" 200 192430
2019-09-20 13:59:20,984 basehttp.log_message 154 INFO    => "GET /account/register/ HTTP/1.1" 200 5589
2019-09-20 13:59:20,997 basehttp.log_message 154 INFO    => "GET /static/js/jquery-3.0.0.min.js HTTP/1.1" 200 86340
2019-09-20 13:59:24,050 basehttp.log_message 154 INFO    => "GET /account/login/ HTTP/1.1" 200 2872
2019-09-20 14:01:11,464 basehttp.log_message 154 INFO    => "GET /account/register/ HTTP/1.1" 200 5589
2019-09-20 14:05:13,004 basehttp.log_message 154 INFO    => "GET /account/register/ HTTP/1.1" 200 5589
2019-09-20 14:05:18,497 basehttp.log_message 154 INFO    => "GET /account/if_username_existed/?username=test1 HTTP/1.1" 200 11
2019-09-20 14:05:30,056 views.new_account  42 INFO    => new account: test1, test@12.com
2019-09-20 14:05:30,154 basehttp.log_message 154 INFO    => "POST /account/new_account/ HTTP/1.1" 200 1127
2019-09-20 14:05:31,351 basehttp.log_message 154 INFO    => "GET /account/login/ HTTP/1.1" 200 2872
2019-09-20 14:05:37,859 views.check_user  20 INFO    => test1 log in
2019-09-20 14:05:37,862 basehttp.log_message 154 INFO    => "POST /account/check_user/ HTTP/1.1" 200 10
2019-09-20 14:05:37,957 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 12333
2019-09-20 14:05:37,975 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap-datetimepicker.zh-CN.js HTTP/1.1" 200 813
2019-09-20 14:05:37,976 basehttp.log_message 154 INFO    => "GET /static/css/datetimepicker.css HTTP/1.1" 200 13490
2019-09-20 14:05:37,977 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap.js HTTP/1.1" 200 131637
2019-09-20 14:05:37,978 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap-datetimepicker.min.css HTTP/1.1" 200 11276
2019-09-20 14:05:37,979 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap-datetimepicker.min.js HTTP/1.1" 200 39213
2019-09-20 14:05:48,329 basehttp.log_message 154 INFO    => "GET /account/logout/ HTTP/1.1" 200 620
2019-09-20 14:05:49,200 basehttp.log_message 154 INFO    => "GET /account/login/ HTTP/1.1" 200 2872
2019-09-20 14:05:55,578 basehttp.log_message 154 INFO    => "GET /account/register/ HTTP/1.1" 200 5589
2019-09-20 14:05:58,684 basehttp.log_message 154 INFO    => "GET /account/if_username_existed/?username=test2 HTTP/1.1" 200 11
2019-09-20 14:06:06,849 views.new_account  42 INFO    => new account: test2, ba@123.com
2019-09-20 14:06:06,932 basehttp.log_message 154 INFO    => "POST /account/new_account/ HTTP/1.1" 200 1127
2019-09-20 14:06:12,588 basehttp.log_message 154 INFO    => "GET /account/login/ HTTP/1.1" 200 2872
2019-09-20 14:06:18,028 views.check_user  20 INFO    => test2 log in
2019-09-20 14:06:18,030 basehttp.log_message 154 INFO    => "POST /account/check_user/ HTTP/1.1" 200 10
2019-09-20 14:06:18,051 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 12333
2019-09-20 14:06:23,624 basehttp.log_message 154 INFO    => "GET /operation/share_space/ HTTP/1.1" 200 9453
2019-09-20 14:06:25,940 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 12333
2019-09-20 14:06:44,250 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap.js.map HTTP/1.1" 200 250568
2019-09-20 14:06:44,263 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap.css.map HTTP/1.1" 200 492048
2019-09-20 14:06:50,123 basehttp.log_message 154 INFO    => "POST /operation/upload_file/ HTTP/1.1" 200 14
2019-09-20 14:06:50,986 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 12959
2019-09-20 14:07:53,019 basehttp.log_message 154 INFO    => "GET /operation/share_file_to_another/?&hashed_name=74b22752a36564a5faec460dfe5b09bf&receiver_username=test1&is_private=0&expiry=2020-09-18%2009:50 HTTP/1.1" 200 15
2019-09-20 14:07:55,328 basehttp.log_message 154 INFO    => "GET /operation/share_space/ HTTP/1.1" 200 9453
2019-09-20 14:07:57,566 basehttp.log_message 154 INFO    => "GET /operation/share_space/ HTTP/1.1" 200 9453
2019-09-20 14:08:00,100 basehttp.log_message 154 INFO    => "GET /account/logout/ HTTP/1.1" 200 620
2019-09-20 14:08:00,878 basehttp.log_message 154 INFO    => "GET /account/login/ HTTP/1.1" 200 2872
2019-09-20 14:08:05,392 views.check_user  20 INFO    => test1 log in
2019-09-20 14:08:05,397 basehttp.log_message 154 INFO    => "POST /account/check_user/ HTTP/1.1" 200 10
2019-09-20 14:08:05,421 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 12333
2019-09-20 14:08:07,033 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 12333
2019-09-20 14:08:08,158 basehttp.log_message 154 INFO    => "GET /operation/share_space/ HTTP/1.1" 200 10285
2019-09-20 14:08:29,985 basehttp.log_message 154 INFO    => "GET /account/logout/ HTTP/1.1" 200 620
2019-09-20 14:08:30,806 basehttp.log_message 154 INFO    => "GET /account/login/ HTTP/1.1" 200 2872
2019-09-20 14:08:39,931 views.check_user  20 INFO    => test1 log in
2019-09-20 14:08:39,933 basehttp.log_message 154 INFO    => "POST /account/check_user/ HTTP/1.1" 200 10
2019-09-20 14:08:39,962 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 12333
2019-09-20 14:08:42,709 basehttp.log_message 154 INFO    => "GET /operation/share_space/ HTTP/1.1" 200 10285
2019-09-20 14:08:44,717 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 12333
2019-09-20 15:36:14,576 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 12333
2019-09-20 15:36:16,699 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 12333
2019-09-20 15:36:17,299 basehttp.log_message 154 INFO    => "GET /operation/share_space/ HTTP/1.1" 200 10285
2019-09-20 15:36:17,982 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 12333
2019-09-20 15:36:18,365 basehttp.log_message 154 INFO    => "GET /operation/share_space/ HTTP/1.1" 200 10285
2019-09-20 15:36:20,211 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 12333
2019-09-20 15:36:21,590 basehttp.log_message 154 INFO    => "GET /operation/share_space/ HTTP/1.1" 200 10285
2019-09-20 15:36:22,399 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 12333
2019-09-20 16:33:52,816 basehttp.log_message 154 INFO    => "GET / HTTP/1.1" 302 0
2019-09-20 16:33:52,824 basehttp.log_message 154 INFO    => "GET /account/login/ HTTP/1.1" 200 2872
2019-09-20 17:39:12,402 basehttp.log_message 154 INFO    => "GET / HTTP/1.1" 302 0
2019-09-20 17:39:12,411 basehttp.log_message 154 INFO    => "GET /account/login/ HTTP/1.1" 200 2872
2019-09-20 17:39:51,076 basehttp.log_message 154 INFO    => "GET /account/register/ HTTP/1.1" 200 5589
2019-09-20 17:39:53,751 basehttp.log_message 154 INFO    => "GET /account/if_username_existed/?username=user1 HTTP/1.1" 200 7
2019-09-20 17:39:56,381 basehttp.log_message 154 INFO    => "GET /account/if_username_existed/?username=user1 HTTP/1.1" 200 7
2019-09-20 17:40:01,092 basehttp.log_message 154 INFO    => "GET /account/if_username_existed/?username=a1 HTTP/1.1" 200 11
2019-09-20 17:40:03,733 basehttp.log_message 154 INFO    => "GET /account/if_username_existed/?username=a HTTP/1.1" 200 11
2019-09-20 17:40:31,262 views.new_account  46 INFO    => new account: a, a@qq.com
2019-09-20 17:40:31,358 basehttp.log_message 154 INFO    => "POST /account/new_account/ HTTP/1.1" 200 1127
2019-09-20 17:40:32,443 basehttp.log_message 154 INFO    => "GET /account/login/ HTTP/1.1" 200 2872
2019-09-20 17:40:34,960 basehttp.log_message 154 INFO    => "GET /account/register/ HTTP/1.1" 200 5589
2019-09-20 17:40:38,791 basehttp.log_message 154 INFO    => "GET /account/if_username_existed/?username=b HTTP/1.1" 200 11
2019-09-20 17:40:50,837 views.new_account  46 INFO    => new account: b, b@qq.com
2019-09-20 17:40:50,922 basehttp.log_message 154 INFO    => "POST /account/new_account/ HTTP/1.1" 200 1127
2019-09-20 17:40:52,382 basehttp.log_message 154 INFO    => "GET /account/login/ HTTP/1.1" 200 2872
2019-09-20 17:40:57,322 views.check_user  24 INFO    => a log in
2019-09-20 17:40:57,325 basehttp.log_message 154 INFO    => "POST /account/check_user/ HTTP/1.1" 200 10
2019-09-20 17:40:57,354 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 12329
2019-09-20 17:41:45,022 views.handle_uploading_file  57 INFO    => uploading-existed-file: 74b22752a36564a5faec460dfe5b09bf
2019-09-20 17:41:45,024 basehttp.log_message 154 INFO    => "POST /operation/upload_file/ HTTP/1.1" 200 14
2019-09-20 17:41:45,894 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 12955
2019-09-20 17:41:54,879 basehttp.log_message 154 INFO    => "POST /operation/upload_file/ HTTP/1.1" 200 14
2019-09-20 17:41:55,707 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 13601
2019-09-20 17:42:10,870 basehttp.log_message 154 INFO    => "GET /account/logout/ HTTP/1.1" 200 620
2019-09-20 17:42:11,704 basehttp.log_message 154 INFO    => "GET /account/login/ HTTP/1.1" 200 2872
2019-09-20 17:42:31,038 basehttp.log_message 154 INFO    => "GET / HTTP/1.1" 302 0
2019-09-20 17:42:31,043 basehttp.log_message 154 INFO    => "GET /account/login/ HTTP/1.1" 200 2872
2019-09-20 17:42:36,281 views.check_user  24 INFO    => b log in
2019-09-20 17:42:36,282 basehttp.log_message 154 INFO    => "POST /account/check_user/ HTTP/1.1" 200 10
2019-09-20 17:42:36,305 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 12329
2019-09-20 17:42:48,591 basehttp.log_message 154 INFO    => "POST /operation/upload_file/ HTTP/1.1" 200 14
2019-09-20 17:42:49,423 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 12963
2019-09-20 17:45:51,895 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 12963
2019-09-20 17:46:39,411 basehttp.log_message 154 INFO    => "POST /operation/upload_file/ HTTP/1.1" 200 14
2019-09-20 17:46:40,246 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 13599
2019-09-20 17:46:45,882 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 13599
2019-09-20 17:47:41,584 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 19080
<<<<<<< HEAD
2019-09-20 17:55:34,482 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 19080
2019-09-20 17:55:36,590 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 19080
2019-09-20 17:55:39,188 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 19080
2019-09-20 17:55:40,221 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=2 HTTP/1.1" 200 13835
2019-09-20 17:55:41,285 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=1 HTTP/1.1" 200 19080
2019-09-20 18:03:23,922 basehttp.log_message 154 INFO    => "GET / HTTP/1.1" 302 0
2019-09-20 18:03:23,925 basehttp.log_message 154 INFO    => "GET /account/login HTTP/1.1" 301 0
2019-09-20 18:03:23,931 basehttp.log_message 154 INFO    => "GET /account/login/ HTTP/1.1" 200 2872
2019-09-20 18:06:03,959 basehttp.log_message 154 INFO    => "GET /account/login/ HTTP/1.1" 200 2872
2019-09-20 18:06:08,294 basehttp.log_message 154 INFO    => "POST /account/check_user/ HTTP/1.1" 200 10
2019-09-20 18:06:12,852 views.check_user  24 INFO    => a log in
2019-09-20 18:06:12,855 basehttp.log_message 154 INFO    => "POST /account/check_user/ HTTP/1.1" 200 10
2019-09-20 18:06:12,923 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 19080
2019-09-20 18:06:22,854 basehttp.log_message 154 INFO    => "POST /operation/upload_file/ HTTP/1.1" 200 14
2019-09-20 18:06:23,696 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 19080
2019-09-20 18:10:29,160 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 19080
2019-09-20 18:10:35,955 log.log_response 228 ERROR   => Internal Server Error: /operation/upload_file/
Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
psycopg2.DataError: value too long for type character varying(30)


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/Users/baiyu/Desktop/BC/black_cloud/src/apps/operation_app/views.py", line 63, in handle_uploading_file
    size=size, file_name=filename, hashed_name=hashed_name)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/db/models/manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/db/models/query.py", line 413, in create
    obj.save(force_insert=True, using=self.db)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/db/models/base.py", line 718, in save
    force_update=force_update, update_fields=update_fields)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/db/models/base.py", line 748, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/db/models/base.py", line 831, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/db/models/base.py", line 869, in _do_insert
    using=using, raw=raw)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/db/models/manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/db/models/query.py", line 1136, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/db/models/sql/compiler.py", line 1289, in execute_sql
    cursor.execute(sql, params)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/db/backends/utils.py", line 100, in execute
    return super().execute(sql, params)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/db/backends/utils.py", line 68, in execute
    return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/db/backends/utils.py", line 77, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/db/utils.py", line 89, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
django.db.utils.DataError: value too long for type character varying(30)

2019-09-20 18:10:35,965 basehttp.log_message 154 ERROR   => "POST /operation/upload_file/ HTTP/1.1" 500 18783
2019-09-20 18:11:27,884 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 19080
2019-09-20 18:11:29,623 basehttp.log_message 154 INFO    => "GET /account/logout/ HTTP/1.1" 200 620
2019-09-20 18:11:30,358 basehttp.log_message 154 INFO    => "GET /account/login/ HTTP/1.1" 200 2872
2019-09-20 18:11:35,831 views.check_user  24 INFO    => b log in
2019-09-20 18:11:35,834 basehttp.log_message 154 INFO    => "POST /account/check_user/ HTTP/1.1" 200 10
2019-09-20 18:11:35,858 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 19080
2019-09-20 18:11:48,112 views.handle_uploading_file  57 INFO    => uploading-existed-file: cceb11223ce713d96401203aae77d229
2019-09-20 18:11:48,116 basehttp.log_message 154 INFO    => "POST /operation/upload_file/ HTTP/1.1" 200 14
2019-09-20 18:11:48,951 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 19080
2019-09-20 18:13:17,954 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=2 HTTP/1.1" 200 15094
2019-09-20 18:13:18,956 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=1 HTTP/1.1" 200 19080
2019-09-20 18:47:30,265 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=1 HTTP/1.1" 200 19080
2019-09-20 18:47:45,740 log.log_response 228 ERROR   => Internal Server Error: /operation/upload_file/
Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/utils.py", line 941, in check_header_validity
    if not pat.match(value):
TypeError: expected string or bytes-like object

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/Users/baiyu/Desktop/BC/black_cloud/src/apps/operation_app/views.py", line 58, in handle_uploading_file
    bk.put_object(filename, upload_file)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/qingstor/sdk/service/bucket.py", line 1530, in put_object
    resp = self.client.send(req.sign())
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/qingstor/sdk/request.py", line 45, in sign
    prepared = self.req.prepare()
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/models.py", line 267, in prepare
    hooks=self.hooks,
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/models.py", line 314, in prepare
    self.prepare_headers(headers)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/models.py", line 448, in prepare_headers
    check_header_validity(header)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/utils.py", line 945, in check_header_validity
    "bytes, not %s" % (name, value, type(value)))
requests.exceptions.InvalidHeader: Value for header {cache-control: ef5864fe58ab506f6ff13031059f4a5e} must be of type str or bytes, not <class 'django.core.files.uploadedfile.InMemoryUploadedFile'>
2019-09-20 18:47:45,742 basehttp.log_message 154 ERROR   => "POST /operation/upload_file/ HTTP/1.1" 500 17114
2019-09-20 18:49:37,535 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=1 HTTP/1.1" 200 19080
2019-09-20 18:49:51,887 log.log_response 228 ERROR   => Internal Server Error: /operation/upload_file/
Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/utils.py", line 941, in check_header_validity
    if not pat.match(value):
TypeError: expected string or bytes-like object

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/Users/baiyu/Desktop/BC/black_cloud/src/apps/operation_app/views.py", line 61, in handle_uploading_file
    bk.put_object(filename, upload_file)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/qingstor/sdk/service/bucket.py", line 1530, in put_object
    resp = self.client.send(req.sign())
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/qingstor/sdk/request.py", line 45, in sign
    prepared = self.req.prepare()
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/models.py", line 267, in prepare
    hooks=self.hooks,
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/models.py", line 314, in prepare
    self.prepare_headers(headers)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/models.py", line 448, in prepare_headers
    check_header_validity(header)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/utils.py", line 945, in check_header_validity
    "bytes, not %s" % (name, value, type(value)))
requests.exceptions.InvalidHeader: Value for header {cache-control: 27263488fe739613a34b30507b4904d1} must be of type str or bytes, not <class 'django.core.files.uploadedfile.InMemoryUploadedFile'>
2019-09-20 18:49:51,890 basehttp.log_message 154 ERROR   => "POST /operation/upload_file/ HTTP/1.1" 500 17112
2019-09-20 18:50:58,696 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=1 HTTP/1.1" 200 19080
2019-09-20 18:51:11,759 log.log_response 228 ERROR   => Internal Server Error: /operation/upload_file/
Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/Users/baiyu/Desktop/BC/black_cloud/src/apps/operation_app/views.py", line 60, in handle_uploading_file
    bk.put_object(upload_file.__dict__.get('file'), upload_file)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/qingstor/sdk/service/bucket.py", line 1528, in put_object
    body=body
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/qingstor/sdk/service/bucket.py", line 1471, in put_object_request
    return Request(self.config, operation)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/qingstor/sdk/request.py", line 31, in __init__
    self.req = Builder(config, operation).parse()
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/qingstor/sdk/build.py", line 48, in parse
    parsed_operation["URI"] = self.parse_request_uri()
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/qingstor/sdk/build.py", line 143, in parse_request_uri
    properties = self.parse_request_properties()
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/qingstor/sdk/build.py", line 138, in parse_request_properties
    parsed_properties[k] = quote(v)
  File "/Users/baiyu/anaconda3/lib/python3.7/urllib/parse.py", line 791, in quote
    return quote_from_bytes(string, safe)
  File "/Users/baiyu/anaconda3/lib/python3.7/urllib/parse.py", line 816, in quote_from_bytes
    raise TypeError("quote_from_bytes() expected bytes")
TypeError: quote_from_bytes() expected bytes
2019-09-20 18:51:11,761 basehttp.log_message 154 ERROR   => "POST /operation/upload_file/ HTTP/1.1" 500 16939
2019-09-20 18:53:39,240 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=1 HTTP/1.1" 200 19080
2019-09-20 18:53:53,833 log.log_response 228 ERROR   => Internal Server Error: /operation/upload_file/
Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/utils.py", line 941, in check_header_validity
    if not pat.match(value):
TypeError: expected string or bytes-like object

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/Users/baiyu/Desktop/BC/black_cloud/src/apps/operation_app/views.py", line 61, in handle_uploading_file
    upload_file.__dict__.get('file'))
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/qingstor/sdk/service/bucket.py", line 1530, in put_object
    resp = self.client.send(req.sign())
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/qingstor/sdk/request.py", line 45, in sign
    prepared = self.req.prepare()
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/models.py", line 267, in prepare
    hooks=self.hooks,
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/models.py", line 314, in prepare
    self.prepare_headers(headers)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/models.py", line 448, in prepare_headers
    check_header_validity(header)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/utils.py", line 945, in check_header_validity
    "bytes, not %s" % (name, value, type(value)))
requests.exceptions.InvalidHeader: Value for header {cache-control: <_io.BytesIO object at 0x102cef620>} must be of type str or bytes, not <class '_io.BytesIO'>
2019-09-20 18:53:53,836 basehttp.log_message 154 ERROR   => "POST /operation/upload_file/ HTTP/1.1" 500 17049
2019-09-20 18:55:27,579 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=1 HTTP/1.1" 200 19080
2019-09-20 18:55:37,029 log.log_response 228 ERROR   => Internal Server Error: /operation/upload_file/
Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/utils.py", line 941, in check_header_validity
    if not pat.match(value):
TypeError: expected string or bytes-like object

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/Users/baiyu/Desktop/BC/black_cloud/src/apps/operation_app/views.py", line 62, in handle_uploading_file
    upload_file.__dict__.get('file'))
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/qingstor/sdk/service/bucket.py", line 1530, in put_object
    resp = self.client.send(req.sign())
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/qingstor/sdk/request.py", line 45, in sign
    prepared = self.req.prepare()
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/models.py", line 267, in prepare
    hooks=self.hooks,
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/models.py", line 314, in prepare
    self.prepare_headers(headers)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/models.py", line 448, in prepare_headers
    check_header_validity(header)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/utils.py", line 945, in check_header_validity
    "bytes, not %s" % (name, value, type(value)))
requests.exceptions.InvalidHeader: Value for header {cache-control: <_io.BytesIO object at 0x102c52678>} must be of type str or bytes, not <class '_io.BytesIO'>
2019-09-20 18:55:37,033 basehttp.log_message 154 ERROR   => "POST /operation/upload_file/ HTTP/1.1" 500 17058
2019-09-20 19:03:36,607 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=1 HTTP/1.1" 200 19080
2019-09-20 19:03:45,376 log.log_response 228 ERROR   => Internal Server Error: /operation/upload_file/
Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/utils.py", line 941, in check_header_validity
    if not pat.match(value):
TypeError: expected string or bytes-like object

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/Users/baiyu/Desktop/BC/black_cloud/src/apps/operation_app/views.py", line 63, in handle_uploading_file
    upload_file.__dict__.get('file'))
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/qingstor/sdk/service/bucket.py", line 1530, in put_object
    resp = self.client.send(req.sign())
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/qingstor/sdk/request.py", line 45, in sign
    prepared = self.req.prepare()
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/models.py", line 267, in prepare
    hooks=self.hooks,
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/models.py", line 314, in prepare
    self.prepare_headers(headers)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/models.py", line 448, in prepare_headers
    check_header_validity(header)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/utils.py", line 945, in check_header_validity
    "bytes, not %s" % (name, value, type(value)))
requests.exceptions.InvalidHeader: Value for header {cache-control: <_io.BytesIO object at 0x11183a780>} must be of type str or bytes, not <class '_io.BytesIO'>
2019-09-20 19:03:45,380 basehttp.log_message 154 ERROR   => "POST /operation/upload_file/ HTTP/1.1" 500 17050
2019-09-20 19:13:03,713 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=1 HTTP/1.1" 200 19080
2019-09-20 19:13:28,554 basehttp.log_message 154 INFO    => "POST /operation/upload_file/ HTTP/1.1" 200 14
2019-09-20 19:13:29,388 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=1 HTTP/1.1" 200 19080
2019-09-20 19:14:56,161 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=1 HTTP/1.1" 200 19080
2019-09-20 19:15:04,458 log.log_response 228 ERROR   => Internal Server Error: /operation/upload_file/
Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/Users/baiyu/Desktop/BC/black_cloud/src/apps/operation_app/views.py", line 68, in handle_uploading_file
    with open('../../storage/'+filename) as f:
FileNotFoundError: [Errno 2] No such file or directory: '../../storage/4月11日问题日志.pages'
2019-09-20 19:15:04,459 basehttp.log_message 154 ERROR   => "POST /operation/upload_file/ HTTP/1.1" 500 15794
2019-09-20 19:21:39,175 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=1 HTTP/1.1" 200 19080
2019-09-20 19:23:09,235 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=1 HTTP/1.1" 200 19080
2019-09-20 19:24:08,768 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=1 HTTP/1.1" 200 19080
2019-09-20 19:24:17,308 log.log_response 228 ERROR   => Internal Server Error: /operation/upload_file/
Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/utils.py", line 941, in check_header_validity
    if not pat.match(value):
TypeError: expected string or bytes-like object

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/Users/baiyu/Desktop/BC/black_cloud/src/apps/operation_app/views.py", line 73, in handle_uploading_file
    bk.put_object(filename, f)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/qingstor/sdk/service/bucket.py", line 1530, in put_object
    resp = self.client.send(req.sign())
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/qingstor/sdk/request.py", line 45, in sign
    prepared = self.req.prepare()
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/models.py", line 267, in prepare
    hooks=self.hooks,
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/models.py", line 314, in prepare
    self.prepare_headers(headers)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/models.py", line 448, in prepare_headers
    check_header_validity(header)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/utils.py", line 945, in check_header_validity
    "bytes, not %s" % (name, value, type(value)))
requests.exceptions.InvalidHeader: Value for header {cache-control: <_io.TextIOWrapper name='/Users/baiyu/Desktop/BC/black_cloud/src/storage/4月13日问题日志.pages' mode='r' encoding='UTF-8'>} must be of type str or bytes, not <class '_io.TextIOWrapper'>
2019-09-20 19:24:17,311 basehttp.log_message 154 ERROR   => "POST /operation/upload_file/ HTTP/1.1" 500 17252
2019-09-20 19:25:23,889 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=1 HTTP/1.1" 200 19080
2019-09-20 19:25:32,602 log.log_response 228 ERROR   => Internal Server Error: /operation/upload_file/
Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/Users/baiyu/Desktop/BC/black_cloud/src/apps/operation_app/views.py", line 72, in handle_uploading_file
    with open(cwd + filename.encode()) as f:
TypeError: can only concatenate str (not "bytes") to str
2019-09-20 19:25:32,603 basehttp.log_message 154 ERROR   => "POST /operation/upload_file/ HTTP/1.1" 500 15706
2019-09-20 19:28:18,203 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=1 HTTP/1.1" 200 19080
2019-09-20 19:28:28,015 log.log_response 228 ERROR   => Internal Server Error: /operation/upload_file/
Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/utils.py", line 941, in check_header_validity
    if not pat.match(value):
TypeError: expected string or bytes-like object

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/Users/baiyu/Desktop/BC/black_cloud/src/apps/operation_app/views.py", line 73, in handle_uploading_file
    bk.put_object(filename, f)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/qingstor/sdk/service/bucket.py", line 1530, in put_object
    resp = self.client.send(req.sign())
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/qingstor/sdk/request.py", line 45, in sign
    prepared = self.req.prepare()
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/models.py", line 267, in prepare
    hooks=self.hooks,
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/models.py", line 314, in prepare
    self.prepare_headers(headers)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/models.py", line 448, in prepare_headers
    check_header_validity(header)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/utils.py", line 945, in check_header_validity
    "bytes, not %s" % (name, value, type(value)))
requests.exceptions.InvalidHeader: Value for header {cache-control: <_io.BufferedReader name='/Users/baiyu/Desktop/BC/black_cloud/src/storage/4月16日问题日志.pages'>} must be of type str or bytes, not <class '_io.BufferedReader'>
2019-09-20 19:28:28,018 basehttp.log_message 154 ERROR   => "POST /operation/upload_file/ HTTP/1.1" 500 17204
2019-09-20 19:30:47,206 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=1 HTTP/1.1" 200 19196
2019-09-20 19:30:54,900 connectionpool.urlopen 748 WARNING => Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x10722ac50>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /_black-cloud
2019-09-20 19:30:56,903 connectionpool.urlopen 748 WARNING => Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x10721d6a0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /_black-cloud
2019-09-20 19:31:00,908 connectionpool.urlopen 748 WARNING => Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x10721dc88>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /_black-cloud
2019-09-20 19:31:00,927 log.log_response 228 ERROR   => Internal Server Error: /operation/upload_file/
Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/urllib3/connection.py", line 157, in _new_conn
    (self._dns_host, self.port), self.timeout, **extra_kw
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/urllib3/util/connection.py", line 61, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "/Users/baiyu/anaconda3/lib/python3.7/socket.py", line 748, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 8] nodename nor servname provided, or not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/urllib3/connectionpool.py", line 672, in urlopen
    chunked=chunked,
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/urllib3/connectionpool.py", line 376, in _make_request
    self._validate_conn(conn)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/urllib3/connectionpool.py", line 994, in _validate_conn
    conn.connect()
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/urllib3/connection.py", line 334, in connect
    conn = self._new_conn()
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/urllib3/connection.py", line 169, in _new_conn
    self, "Failed to establish a new connection: %s" % e
urllib3.exceptions.NewConnectionError: <urllib3.connection.VerifiedHTTPSConnection object at 0x10721d0f0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/adapters.py", line 449, in send
    timeout=timeout
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/urllib3/connectionpool.py", line 762, in urlopen
    **response_kw
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/urllib3/connectionpool.py", line 762, in urlopen
    **response_kw
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/urllib3/connectionpool.py", line 762, in urlopen
    **response_kw
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/urllib3/connectionpool.py", line 720, in urlopen
    method, url, error=e, _pool=self, _stacktrace=sys.exc_info()[2]
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/urllib3/util/retry.py", line 436, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='pek3b.qingstor.com', port=443): Max retries exceeded with url: /_black-cloud (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x10721d0f0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/Users/baiyu/Desktop/BC/black_cloud/src/apps/operation_app/views.py", line 67, in handle_uploading_file
    bk = new_bucket()
  File "/Users/baiyu/Desktop/BC/black_cloud/src/storage/qingstor_package.py", line 10, in new_bucket
    bucket.put()
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/qingstor/sdk/service/bucket.py", line 550, in put
    resp = self.client.send(req.sign())
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/adapters.py", line 516, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='pek3b.qingstor.com', port=443): Max retries exceeded with url: /_black-cloud (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x10721d0f0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known'))
2019-09-20 19:31:00,931 basehttp.log_message 154 ERROR   => "POST /operation/upload_file/ HTTP/1.1" 500 19725
2019-09-20 19:32:26,710 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=1 HTTP/1.1" 200 19196
2019-09-20 19:32:37,263 log.log_response 228 ERROR   => Internal Server Error: /operation/upload_file/
Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/utils.py", line 941, in check_header_validity
    if not pat.match(value):
TypeError: expected string or bytes-like object

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/Users/baiyu/Desktop/BC/black_cloud/src/apps/operation_app/views.py", line 73, in handle_uploading_file
    bk.put_object(hashed_name, f)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/qingstor/sdk/service/bucket.py", line 1530, in put_object
    resp = self.client.send(req.sign())
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/qingstor/sdk/request.py", line 45, in sign
    prepared = self.req.prepare()
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/models.py", line 267, in prepare
    hooks=self.hooks,
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/models.py", line 314, in prepare
    self.prepare_headers(headers)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/models.py", line 448, in prepare_headers
    check_header_validity(header)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/utils.py", line 945, in check_header_validity
    "bytes, not %s" % (name, value, type(value)))
requests.exceptions.InvalidHeader: Value for header {cache-control: <_io.BufferedReader name='/Users/baiyu/Desktop/BC/black_cloud/src/storage/4月17日问题日志.pages'>} must be of type str or bytes, not <class '_io.BufferedReader'>
2019-09-20 19:32:37,266 basehttp.log_message 154 ERROR   => "POST /operation/upload_file/ HTTP/1.1" 500 17207
2019-09-20 19:34:02,127 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=1 HTTP/1.1" 200 19196
2019-09-20 19:34:15,424 log.log_response 228 ERROR   => Internal Server Error: /operation/upload_file/
Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/utils.py", line 941, in check_header_validity
    if not pat.match(value):
TypeError: expected string or bytes-like object

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/Users/baiyu/Desktop/BC/black_cloud/src/apps/operation_app/views.py", line 73, in handle_uploading_file
    bk.put_object(hashed_name, f)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/qingstor/sdk/service/bucket.py", line 1530, in put_object
    resp = self.client.send(req.sign())
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/qingstor/sdk/request.py", line 45, in sign
    prepared = self.req.prepare()
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/models.py", line 267, in prepare
    hooks=self.hooks,
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/models.py", line 314, in prepare
    self.prepare_headers(headers)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/models.py", line 448, in prepare_headers
    check_header_validity(header)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/requests/utils.py", line 945, in check_header_validity
    "bytes, not %s" % (name, value, type(value)))
requests.exceptions.InvalidHeader: Value for header {cache-control: <_io.BufferedReader name='/Users/baiyu/Desktop/BC/black_cloud/src/storage/fenghuo.pdf'>} must be of type str or bytes, not <class '_io.BufferedReader'>
2019-09-20 19:34:15,430 basehttp.log_message 154 ERROR   => "POST /operation/upload_file/ HTTP/1.1" 500 17143
2019-09-20 19:36:13,185 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=1 HTTP/1.1" 200 19196
2019-09-20 19:36:21,558 basehttp.log_message 154 INFO    => "POST /operation/upload_file/ HTTP/1.1" 200 14
2019-09-20 19:36:22,389 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=1 HTTP/1.1" 200 19196
2019-09-20 19:38:21,614 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=1 HTTP/1.1" 200 19196
2019-09-20 19:38:31,152 basehttp.log_message 154 INFO    => "POST /operation/upload_file/ HTTP/1.1" 200 14
2019-09-20 19:38:31,985 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=1 HTTP/1.1" 200 19196
2019-09-20 19:39:24,290 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=1 HTTP/1.1" 200 19196
2019-09-20 19:39:33,171 basehttp.log_message 154 INFO    => "POST /operation/upload_file/ HTTP/1.1" 200 14
2019-09-20 19:39:34,004 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=1 HTTP/1.1" 200 19196
2019-09-20 19:40:16,936 log.log_response 228 ERROR   => Internal Server Error: /operation/upload_file/
Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/Users/baiyu/Desktop/BC/black_cloud/src/apps/operation_app/views.py", line 71, in handle_uploading_file
    with open(cwd + filename, 'rb') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/Users/baiyu/Desktop/BC/black_cloud/src/storage/负载均衡 相关参数的记录'
2019-09-20 19:40:16,937 basehttp.log_message 154 ERROR   => "POST /operation/upload_file/ HTTP/1.1" 500 15865
2019-09-20 19:41:56,736 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=1 HTTP/1.1" 200 19196
2019-09-20 19:42:06,585 basehttp.log_message 154 INFO    => "POST /operation/upload_file/ HTTP/1.1" 200 14
2019-09-20 19:42:07,419 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=1 HTTP/1.1" 200 19196
=======
2019-09-20 19:12:16,780 basehttp.log_message 154 INFO    => "GET / HTTP/1.1" 302 0
2019-09-20 19:12:16,947 basehttp.log_message 154 INFO    => "GET /account/login/ HTTP/1.1" 200 2872
2019-09-20 19:12:17,098 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap.css HTTP/1.1" 304 0
>>>>>>> dd0dce1428d36cc4b60b5b84c799dff8c3630035
2019-09-20 21:23:24,831 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=1 HTTP/1.1" 302 0
2019-09-20 21:23:24,834 basehttp.log_message 154 INFO    => "GET /account/login?next=/operation/operation_board/%3Fcurrent_page%3D1 HTTP/1.1" 301 0
2019-09-20 21:23:24,844 basehttp.log_message 154 INFO    => "GET /account/login/?next=/operation/operation_board/%3Fcurrent_page%3D1 HTTP/1.1" 200 2872
2019-09-20 21:23:29,955 basehttp.log_message 154 INFO    => "POST /account/check_user/ HTTP/1.1" 200 10
2019-09-20 21:23:34,031 basehttp.log_message 154 INFO    => "POST /account/check_user/ HTTP/1.1" 200 10
2019-09-20 21:23:40,827 basehttp.log_message 154 INFO    => "POST /account/check_user/ HTTP/1.1" 200 10
2019-09-20 21:23:42,505 basehttp.log_message 154 INFO    => "GET /account/register/ HTTP/1.1" 200 5589
2019-09-20 21:23:44,431 basehttp.log_message 154 INFO    => "GET /account/if_username_existed/?username=a HTTP/1.1" 200 11
2019-09-20 21:24:04,297 views.new_account  46 INFO    => new account: a, 1234@qq.com
2019-09-20 21:24:04,783 basehttp.log_message 154 INFO    => "POST /account/new_account/ HTTP/1.1" 200 1127
2019-09-20 21:24:05,681 basehttp.log_message 154 INFO    => "GET /account/login/ HTTP/1.1" 200 2872
2019-09-20 21:24:11,127 views.check_user  24 INFO    => a log in
2019-09-20 21:24:11,266 basehttp.log_message 154 INFO    => "POST /account/check_user/ HTTP/1.1" 200 10
2019-09-20 21:24:11,742 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 12979
2019-09-20 21:31:37,489 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 13331
2019-09-20 21:31:57,215 basehttp.log_message 154 INFO    => "POST /operation/upload_file/ HTTP/1.1" 200 14
2019-09-20 21:31:58,487 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 14172
2019-09-20 21:32:17,553 log.log_response 228 ERROR   => Internal Server Error: /operation/view_file/
Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/Users/baiyu/Desktop/BC/black_cloud/src/apps/operation_app/views.py", line 162, in view_file
    filename = request.get('file_name')
AttributeError: 'WSGIRequest' object has no attribute 'get'
2019-09-20 21:32:17,555 basehttp.log_message 154 ERROR   => "GET /operation/view_file/?file_name=default.profraw&hashed_name=d41d8cd98f00b204e9800998ecf8427e&file_type=octet-stream HTTP/1.1" 500 72073
2019-09-20 21:34:42,502 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 14172
2019-09-20 21:34:45,451 log.log_response 228 ERROR   => Internal Server Error: /operation/view_file/
Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/Users/baiyu/Desktop/BC/black_cloud/src/apps/operation_app/views.py", line 162, in view_file
    filename = request.get('file_name')
AttributeError: 'WSGIRequest' object has no attribute 'get'
2019-09-20 21:34:45,452 basehttp.log_message 154 ERROR   => "GET /operation/view_file/?file_name=4%E6%9C%8811%E6%97%A5%E9%97%AE%E9%A2%98%E6%97%A5%E5%BF%97.pages&hashed_name=4%E6%9C%8811%E6%97%A5%E9%97%AE%E9%A2%98%E6%97%A5%E5%BF%97.pages&file_type=x-iwork-pages-sffpages HTTP/1.1" 500 72802
2019-09-20 21:37:03,887 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 14172
2019-09-20 21:37:13,319 log.log_response 228 ERROR   => Internal Server Error: /operation/view_file/
Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/utils/deprecation.py", line 93, in __call__
    response = self.process_response(request, response)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/middleware/clickjacking.py", line 26, in process_response
    if response.get('X-Frame-Options') is not None:
AttributeError: 'str' object has no attribute 'get'
2019-09-20 21:37:13,320 basehttp.log_message 154 ERROR   => "GET /operation/view_file/?file_name=4%E6%9C%8811%E6%97%A5%E9%97%AE%E9%A2%98%E6%97%A5%E5%BF%97.pages&hashed_name=4%E6%9C%8811%E6%97%A5%E9%97%AE%E9%A2%98%E6%97%A5%E5%BF%97.pages&file_type=x-iwork-pages-sffpages HTTP/1.1" 500 60399
2019-09-20 21:40:43,512 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 14172
2019-09-20 21:40:57,643 basehttp.log_message 154 INFO    => "POST /operation/upload_file/ HTTP/1.1" 200 14
2019-09-20 21:40:58,897 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 14944
2019-09-20 21:41:51,093 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 14944
2019-09-20 21:42:06,517 basehttp.log_message 154 INFO    => "POST /operation/upload_file/ HTTP/1.1" 200 14
2019-09-20 21:42:07,769 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 15865
2019-09-20 21:42:13,625 log.log_response 228 ERROR   => Internal Server Error: /operation/view_file/
Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/utils/deprecation.py", line 93, in __call__
    response = self.process_response(request, response)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/middleware/clickjacking.py", line 26, in process_response
    if response.get('X-Frame-Options') is not None:
AttributeError: 'str' object has no attribute 'get'
2019-09-20 21:42:13,626 basehttp.log_message 154 ERROR   => "GET /operation/view_file/?file_name=%E8%B5%84%E6%BA%90%E7%BB%91%E5%AE%9Aeip%E4%BB%A3%E7%A0%81%E9%80%BB%E8%BE%91%E5%9B%BE-%E6%9C%80%E7%BB%88%E4%BB%A3%E7%A0%81%E7%89%88.png&hashed_name=%E8%B5%84%E6%BA%90%E7%BB%91%E5%AE%9Aeip%E4%BB%A3%E7%A0%81%E9%80%BB%E8%BE%91%E5%9B%BE-%E6%9C%80%E7%BB%88%E4%BB%A3%E7%A0%81%E7%89%88.png&file_type=png HTTP/1.1" 500 61164
2019-09-20 21:43:29,857 basehttp.log_message 154 INFO    => "POST /operation/upload_file/ HTTP/1.1" 200 14
2019-09-20 21:43:31,143 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 16786
2019-09-20 21:43:45,383 log.log_response 228 ERROR   => Internal Server Error: /operation/view_file/
Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/utils/deprecation.py", line 93, in __call__
    response = self.process_response(request, response)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/middleware/clickjacking.py", line 26, in process_response
    if response.get('X-Frame-Options') is not None:
AttributeError: 'str' object has no attribute 'get'
2019-09-20 21:43:45,384 basehttp.log_message 154 ERROR   => "GET /operation/view_file/?file_name=%E8%B5%84%E6%BA%90%E7%BB%91%E5%AE%9Aeip%E4%BB%A3%E7%A0%81%E9%80%BB%E8%BE%91%E5%9B%BE-%E6%9C%80%E7%BB%88%E4%BB%A3%E7%A0%81%E7%89%88.png&hashed_name=%E8%B5%84%E6%BA%90%E7%BB%91%E5%AE%9Aeip%E4%BB%A3%E7%A0%81%E9%80%BB%E8%BE%91%E5%9B%BE-%E6%9C%80%E7%BB%88%E4%BB%A3%E7%A0%81%E7%89%88.png&file_type=png HTTP/1.1" 500 61164
2019-09-20 21:46:37,544 basehttp.log_message 154 INFO    => "POST /operation/upload_file/ HTTP/1.1" 200 14
2019-09-20 21:46:38,832 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 17472
2019-09-20 21:46:44,123 log.log_response 228 ERROR   => Internal Server Error: /operation/view_file/
Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/utils/deprecation.py", line 93, in __call__
    response = self.process_response(request, response)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/middleware/clickjacking.py", line 26, in process_response
    if response.get('X-Frame-Options') is not None:
AttributeError: 'str' object has no attribute 'get'
2019-09-20 21:46:44,124 basehttp.log_message 154 ERROR   => "GET /operation/view_file/?file_name=fenghuo.pdf&hashed_name=fenghuo.pdf&file_type=pdf HTTP/1.1" 500 59610
2019-09-20 21:53:44,714 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 17472
2019-09-20 21:53:58,317 basehttp.log_message 154 INFO    => "POST /operation/upload_file/ HTTP/1.1" 200 14
2019-09-20 21:53:59,573 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 18244
2019-09-20 21:54:05,477 log.log_response 228 ERROR   => Internal Server Error: /operation/view_file/
Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/utils/deprecation.py", line 93, in __call__
    response = self.process_response(request, response)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/middleware/clickjacking.py", line 26, in process_response
    if response.get('X-Frame-Options') is not None:
AttributeError: 'str' object has no attribute 'get'
2019-09-20 21:54:05,478 basehttp.log_message 154 ERROR   => "GET /operation/view_file/?file_name=%E5%8F%96%E4%B8%8D%E5%88%B0SSD%E4%BD%BF%E7%94%A8%E7%8E%87.png&hashed_name=%E5%8F%96%E4%B8%8D%E5%88%B0SSD%E4%BD%BF%E7%94%A8%E7%8E%87.png&file_type=png HTTP/1.1" 500 60238
2019-09-20 22:12:10,267 log.log_response 228 ERROR   => Internal Server Error: /operation/view_file/
Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/utils/deprecation.py", line 93, in __call__
    response = self.process_response(request, response)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/middleware/clickjacking.py", line 26, in process_response
    if response.get('X-Frame-Options') is not None:
AttributeError: 'str' object has no attribute 'get'
2019-09-20 22:12:10,268 basehttp.log_message 154 ERROR   => "GET /operation/view_file/?file_name=%E5%8F%96%E4%B8%8D%E5%88%B0SSD%E4%BD%BF%E7%94%A8%E7%8E%87.png&hashed_name=%E5%8F%96%E4%B8%8D%E5%88%B0SSD%E4%BD%BF%E7%94%A8%E7%8E%87.png&file_type=png HTTP/1.1" 500 60550
2019-09-20 22:13:51,690 log.log_response 228 ERROR   => Internal Server Error: /operation/view_file/
Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/utils/deprecation.py", line 93, in __call__
    response = self.process_response(request, response)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/middleware/clickjacking.py", line 26, in process_response
    if response.get('X-Frame-Options') is not None:
AttributeError: 'str' object has no attribute 'get'
2019-09-20 22:13:51,692 basehttp.log_message 154 ERROR   => "GET /operation/view_file/?file_name=%E5%8F%96%E4%B8%8D%E5%88%B0SSD%E4%BD%BF%E7%94%A8%E7%8E%87.png&hashed_name=%E5%8F%96%E4%B8%8D%E5%88%B0SSD%E4%BD%BF%E7%94%A8%E7%8E%87.png&file_type=png HTTP/1.1" 500 60590
2019-09-20 22:20:10,013 log.log_response 228 ERROR   => Internal Server Error: /operation/view_file/
Traceback (most recent call last):
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/utils/deprecation.py", line 93, in __call__
    response = self.process_response(request, response)
  File "/Users/baiyu/Desktop/BC/bc_env/lib/python3.7/site-packages/django/middleware/clickjacking.py", line 26, in process_response
    if response.get('X-Frame-Options') is not None:
AttributeError: 'str' object has no attribute 'get'
2019-09-20 22:20:10,014 basehttp.log_message 154 ERROR   => "GET /operation/view_file/?file_name=%E5%8F%96%E4%B8%8D%E5%88%B0SSD%E4%BD%BF%E7%94%A8%E7%8E%87.png&hashed_name=%E5%8F%96%E4%B8%8D%E5%88%B0SSD%E4%BD%BF%E7%94%A8%E7%8E%87.png&file_type=png HTTP/1.1" 500 60590
2019-09-20 22:21:54,075 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 18244
2019-09-20 22:21:56,623 basehttp.log_message 154 INFO    => "GET /operation/view_file/?file_name=%E5%8F%96%E4%B8%8D%E5%88%B0SSD%E4%BD%BF%E7%94%A8%E7%8E%87.png&hashed_name=%E5%8F%96%E4%B8%8D%E5%88%B0SSD%E4%BD%BF%E7%94%A8%E7%8E%87.png&file_type=png HTTP/1.1" 200 175
2019-09-20 22:23:01,208 basehttp.log_message 154 INFO    => "GET /operation/view_file/?file_name=%E5%8F%96%E4%B8%8D%E5%88%B0SSD%E4%BD%BF%E7%94%A8%E7%8E%87.png&hashed_name=%E5%8F%96%E4%B8%8D%E5%88%B0SSD%E4%BD%BF%E7%94%A8%E7%8E%87.png&file_type=png HTTP/1.1" 200 173
2019-09-20 22:24:07,511 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 18244
2019-09-20 22:24:15,357 basehttp.log_message 154 INFO    => "GET /operation/view_file/?file_name=fenghuo.pdf&hashed_name=fenghuo.pdf&file_type=pdf HTTP/1.1" 200 159
2019-09-20 22:24:58,461 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 18244
2019-09-20 22:24:58,495 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap-datetimepicker.zh-CN.js HTTP/1.1" 200 813
2019-09-20 22:24:58,497 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap-datetimepicker.min.js HTTP/1.1" 200 39213
2019-09-20 22:24:58,498 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap-datetimepicker.min.css HTTP/1.1" 200 11276
2019-09-20 22:24:58,503 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap.js HTTP/1.1" 200 131637
2019-09-20 22:24:58,504 basehttp.log_message 154 INFO    => "GET /static/css/datetimepicker.css HTTP/1.1" 200 13490
2019-09-20 22:24:58,506 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap.css HTTP/1.1" 200 192430
2019-09-20 22:24:58,507 basehttp.log_message 154 INFO    => "GET /static/js/jquery-3.0.0.min.js HTTP/1.1" 200 86340
2019-09-20 22:24:58,549 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap.js.map HTTP/1.1" 200 250568
2019-09-20 22:24:58,615 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap.css.map HTTP/1.1" 200 492048
2019-09-20 22:35:42,561 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 302 0
2019-09-20 22:35:42,564 basehttp.log_message 154 INFO    => "GET /account/login?next=/operation/operation_board/ HTTP/1.1" 301 0
2019-09-20 22:35:42,574 basehttp.log_message 154 INFO    => "GET /account/login/?next=/operation/operation_board/ HTTP/1.1" 200 2876
2019-09-20 22:35:42,594 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap.css HTTP/1.1" 200 192430
2019-09-20 22:35:42,648 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap.css.map HTTP/1.1" 200 492048
2019-09-20 22:35:50,605 basehttp.log_message 154 INFO    => "POST /account/check_user/ HTTP/1.1" 200 10
2019-09-20 22:35:55,503 basehttp.log_message 154 INFO    => "POST /account/check_user/ HTTP/1.1" 200 10
2019-09-20 22:35:58,156 views.check_user  24 INFO    => b log in
2019-09-20 22:35:58,158 basehttp.log_message 154 INFO    => "POST /account/check_user/ HTTP/1.1" 200 10
2019-09-20 22:35:58,230 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 22033
2019-09-20 22:35:58,265 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap-datetimepicker.zh-CN.js HTTP/1.1" 200 813
2019-09-20 22:35:58,269 basehttp.log_message 154 INFO    => "GET /static/js/jquery-3.0.0.min.js HTTP/1.1" 200 86340
2019-09-20 22:35:58,270 basehttp.log_message 154 INFO    => "GET /static/css/datetimepicker.css HTTP/1.1" 200 13490
2019-09-20 22:35:58,270 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap-datetimepicker.min.js HTTP/1.1" 200 39213
2019-09-20 22:35:58,272 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap.css HTTP/1.1" 200 192430
2019-09-20 22:35:58,274 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap-datetimepicker.min.css HTTP/1.1" 200 11276
2019-09-20 22:35:58,276 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap.js HTTP/1.1" 200 131637
2019-09-20 22:35:58,339 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap.js.map HTTP/1.1" 200 250568
2019-09-20 22:35:58,402 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap.css.map HTTP/1.1" 200 492048
2019-09-20 22:36:16,388 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=2 HTTP/1.1" 200 22364
2019-09-20 22:36:16,418 basehttp.log_message 154 INFO    => "GET /static/css/datetimepicker.css HTTP/1.1" 200 13490
2019-09-20 22:36:16,419 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap-datetimepicker.zh-CN.js HTTP/1.1" 200 813
2019-09-20 22:36:16,421 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap.css HTTP/1.1" 200 192430
2019-09-20 22:36:16,426 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap.js HTTP/1.1" 200 131637
2019-09-20 22:36:16,425 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap-datetimepicker.min.js HTTP/1.1" 200 39213
2019-09-20 22:36:16,422 basehttp.log_message 154 INFO    => "GET /static/js/jquery-3.0.0.min.js HTTP/1.1" 200 86340
2019-09-20 22:36:16,428 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap-datetimepicker.min.css HTTP/1.1" 200 11276
2019-09-20 22:36:16,513 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap.js.map HTTP/1.1" 200 250568
2019-09-20 22:36:16,539 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap.css.map HTTP/1.1" 200 492048
2019-09-20 22:36:17,521 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=3 HTTP/1.1" 200 21543
2019-09-20 22:36:17,549 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap.js HTTP/1.1" 200 131637
2019-09-20 22:36:17,550 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap-datetimepicker.min.js HTTP/1.1" 200 39213
2019-09-20 22:36:17,551 basehttp.log_message 154 INFO    => "GET /static/css/datetimepicker.css HTTP/1.1" 200 13490
2019-09-20 22:36:17,552 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap-datetimepicker.zh-CN.js HTTP/1.1" 200 813
2019-09-20 22:36:17,553 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap.css HTTP/1.1" 200 192430
2019-09-20 22:36:17,555 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap-datetimepicker.min.css HTTP/1.1" 200 11276
2019-09-20 22:36:17,556 basehttp.log_message 154 INFO    => "GET /static/js/jquery-3.0.0.min.js HTTP/1.1" 200 86340
2019-09-20 22:36:17,609 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap.js.map HTTP/1.1" 200 250568
2019-09-20 22:36:17,624 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap.css.map HTTP/1.1" 200 492048
2019-09-20 22:37:10,328 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=3 HTTP/1.1" 200 21549
2019-09-20 22:37:10,370 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap-datetimepicker.zh-CN.js HTTP/1.1" 200 813
2019-09-20 22:37:10,376 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap-datetimepicker.min.js HTTP/1.1" 200 39213
2019-09-20 22:37:10,378 basehttp.log_message 154 INFO    => "GET /static/css/datetimepicker.css HTTP/1.1" 200 13490
2019-09-20 22:37:10,379 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap-datetimepicker.min.css HTTP/1.1" 200 11276
2019-09-20 22:37:10,380 basehttp.log_message 154 INFO    => "GET /static/js/jquery-3.0.0.min.js HTTP/1.1" 200 86340
2019-09-20 22:37:10,381 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap.css HTTP/1.1" 200 192430
2019-09-20 22:37:10,383 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap.js HTTP/1.1" 200 131637
2019-09-20 22:37:10,429 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap.js.map HTTP/1.1" 200 250568
2019-09-20 22:37:10,662 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap.css.map HTTP/1.1" 200 492048
2019-09-20 22:38:32,431 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=3 HTTP/1.1" 200 21586
2019-09-20 22:38:32,461 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap-datetimepicker.zh-CN.js HTTP/1.1" 200 813
2019-09-20 22:38:32,461 basehttp.log_message 154 INFO    => "GET /static/js/jquery-3.0.0.min.js HTTP/1.1" 200 86340
2019-09-20 22:38:32,464 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap.js HTTP/1.1" 200 131637
2019-09-20 22:38:32,467 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap-datetimepicker.min.js HTTP/1.1" 200 39213
2019-09-20 22:38:32,468 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap-datetimepicker.min.css HTTP/1.1" 200 11276
2019-09-20 22:38:32,474 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap.css HTTP/1.1" 200 192430
2019-09-20 22:38:32,475 basehttp.log_message 154 INFO    => "GET /static/css/datetimepicker.css HTTP/1.1" 200 13490
2019-09-20 22:38:32,519 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap.js.map HTTP/1.1" 200 250568
2019-09-20 22:38:32,592 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap.css.map HTTP/1.1" 200 492048
2019-09-20 22:40:25,654 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=3 HTTP/1.1" 200 21632
2019-09-20 22:40:25,683 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap.js HTTP/1.1" 200 131637
2019-09-20 22:40:25,687 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap-datetimepicker.min.css HTTP/1.1" 200 11276
2019-09-20 22:40:25,692 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap-datetimepicker.zh-CN.js HTTP/1.1" 200 813
2019-09-20 22:40:25,694 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap-datetimepicker.min.js HTTP/1.1" 200 39213
2019-09-20 22:40:25,696 basehttp.log_message 154 INFO    => "GET /static/css/datetimepicker.css HTTP/1.1" 200 13490
2019-09-20 22:40:25,697 basehttp.log_message 154 INFO    => "GET /static/js/jquery-3.0.0.min.js HTTP/1.1" 200 86340
2019-09-20 22:40:25,698 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap.css HTTP/1.1" 200 192430
2019-09-20 22:40:25,755 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap.js.map HTTP/1.1" 200 250568
2019-09-20 22:40:25,792 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap.css.map HTTP/1.1" 200 492048
2019-09-20 22:42:06,543 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=3 HTTP/1.1" 200 21654
2019-09-20 22:42:06,559 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap.js HTTP/1.1" 200 131637
2019-09-20 22:42:06,565 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap-datetimepicker.min.js HTTP/1.1" 200 39213
2019-09-20 22:42:06,565 basehttp.log_message 154 INFO    => "GET /static/js/jquery-3.0.0.min.js HTTP/1.1" 200 86340
2019-09-20 22:42:06,574 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap-datetimepicker.min.css HTTP/1.1" 200 11276
2019-09-20 22:42:06,575 basehttp.log_message 154 INFO    => "GET /static/css/datetimepicker.css HTTP/1.1" 200 13490
2019-09-20 22:42:06,576 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap-datetimepicker.zh-CN.js HTTP/1.1" 200 813
2019-09-20 22:42:06,580 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap.css HTTP/1.1" 200 192430
2019-09-20 22:42:06,601 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap.js.map HTTP/1.1" 200 250568
2019-09-20 22:42:06,646 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap.css.map HTTP/1.1" 200 492048
2019-09-20 22:43:14,709 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=3 HTTP/1.1" 200 21688
2019-09-20 22:43:14,734 basehttp.log_message 154 INFO    => "GET /static/css/datetimepicker.css HTTP/1.1" 200 13490
2019-09-20 22:43:14,735 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap-datetimepicker.zh-CN.js HTTP/1.1" 200 813
2019-09-20 22:43:14,737 basehttp.log_message 154 INFO    => "GET /static/js/jquery-3.0.0.min.js HTTP/1.1" 200 86340
2019-09-20 22:43:14,740 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap-datetimepicker.min.css HTTP/1.1" 200 11276
2019-09-20 22:43:14,740 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap.css HTTP/1.1" 200 192430
2019-09-20 22:43:14,740 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap-datetimepicker.min.js HTTP/1.1" 200 39213
2019-09-20 22:43:14,745 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap.js HTTP/1.1" 200 131637
2019-09-20 22:43:14,791 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap.js.map HTTP/1.1" 200 250568
2019-09-20 22:43:14,827 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap.css.map HTTP/1.1" 200 492048
2019-09-20 22:44:39,364 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=3 HTTP/1.1" 200 21696
2019-09-20 22:44:39,392 basehttp.log_message 154 INFO    => "GET /static/js/jquery-3.0.0.min.js HTTP/1.1" 200 86340
2019-09-20 22:44:39,397 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap-datetimepicker.min.js HTTP/1.1" 200 39213
2019-09-20 22:44:39,400 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap-datetimepicker.zh-CN.js HTTP/1.1" 200 813
2019-09-20 22:44:39,401 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap.js HTTP/1.1" 200 131637
2019-09-20 22:44:39,401 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap-datetimepicker.min.css HTTP/1.1" 200 11276
2019-09-20 22:44:39,402 basehttp.log_message 154 INFO    => "GET /static/css/datetimepicker.css HTTP/1.1" 200 13490
2019-09-20 22:44:39,409 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap.css HTTP/1.1" 200 192430
2019-09-20 22:44:39,439 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap.js.map HTTP/1.1" 200 250568
2019-09-20 22:44:39,481 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap.css.map HTTP/1.1" 200 492048
2019-09-20 22:44:41,569 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=3 HTTP/1.1" 200 21696
2019-09-20 22:44:41,584 basehttp.log_message 154 INFO    => "GET /static/js/jquery-3.0.0.min.js HTTP/1.1" 200 86340
2019-09-20 22:44:41,592 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap.js HTTP/1.1" 200 131637
2019-09-20 22:44:41,593 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap-datetimepicker.min.css HTTP/1.1" 200 11276
2019-09-20 22:44:41,594 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap-datetimepicker.zh-CN.js HTTP/1.1" 200 813
2019-09-20 22:44:41,595 basehttp.log_message 154 INFO    => "GET /static/css/datetimepicker.css HTTP/1.1" 200 13490
2019-09-20 22:44:41,596 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap-datetimepicker.min.js HTTP/1.1" 200 39213
2019-09-20 22:44:41,599 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap.css HTTP/1.1" 200 192430
2019-09-20 22:44:41,627 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap.js.map HTTP/1.1" 200 250568
2019-09-20 22:44:41,662 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap.css.map HTTP/1.1" 200 492048
2019-09-20 22:44:46,516 basehttp.log_message 154 INFO    => "GET /operation/view_file/?file_name=fenghuo.pdf&hashed_name=fenghuo.pdf&file_type=pdf HTTP/1.1" 200 159
2019-09-20 22:45:01,477 basehttp.log_message 154 INFO    => "GET /operation/view_file/?file_name=fenghuo.pdf&hashed_name=fenghuo.pdf&file_type=pdf HTTP/1.1" 200 163
2019-09-20 22:45:33,990 basehttp.log_message 154 INFO    => "POST /operation/upload_file/ HTTP/1.1" 200 14
2019-09-20 22:45:34,822 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=3 HTTP/1.1" 200 22373
2019-09-20 22:45:34,844 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap-datetimepicker.zh-CN.js HTTP/1.1" 200 813
2019-09-20 22:45:34,845 basehttp.log_message 154 INFO    => "GET /static/css/datetimepicker.css HTTP/1.1" 200 13490
2019-09-20 22:45:34,853 basehttp.log_message 154 INFO    => "GET /static/js/jquery-3.0.0.min.js HTTP/1.1" 200 86340
2019-09-20 22:45:34,853 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap-datetimepicker.min.js HTTP/1.1" 200 39213
2019-09-20 22:45:34,855 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap-datetimepicker.min.css HTTP/1.1" 200 11276
2019-09-20 22:45:34,855 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap.css HTTP/1.1" 200 192430
2019-09-20 22:45:34,860 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap.js HTTP/1.1" 200 131637
2019-09-20 22:45:34,911 basehttp.log_message 154 INFO    => "GET /static/js/bootstrap.js.map HTTP/1.1" 200 250568
2019-09-20 22:45:34,939 basehttp.log_message 154 INFO    => "GET /static/css/bootstrap.css.map HTTP/1.1" 200 492048
2019-09-20 22:45:44,378 basehttp.log_message 154 INFO    => "GET /operation/view_file/?file_name=video.mp4&hashed_name=video.mp4&file_type=mp4 HTTP/1.1" 200 157
2019-09-20 23:02:16,768 basehttp.log_message 154 INFO    => "GET /account/login/ HTTP/1.1" 200 2876
2019-09-20 23:02:22,261 basehttp.log_message 154 INFO    => "POST /account/check_user/ HTTP/1.1" 200 10
2019-09-20 23:02:25,324 views.check_user  24 INFO    => b log in
2019-09-20 23:02:25,326 basehttp.log_message 154 INFO    => "POST /account/check_user/ HTTP/1.1" 200 10
2019-09-20 23:02:25,393 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 22185
2019-09-20 23:02:45,359 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=2 HTTP/1.1" 200 22516
2019-09-20 23:02:46,651 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=3 HTTP/1.1" 200 22372
2019-09-20 23:02:49,848 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=1 HTTP/1.1" 200 22185
2019-09-20 23:11:23,287 basehttp.log_message 154 INFO    => "GET /operation/share_space/ HTTP/1.1" 200 9435
2019-09-20 23:11:25,030 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 22185
2019-09-20 23:11:26,236 basehttp.log_message 154 INFO    => "GET /operation/share_space/ HTTP/1.1" 200 9435
2019-09-20 23:11:29,560 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 22185
2019-09-20 23:11:30,334 basehttp.log_message 154 INFO    => "GET /operation/share_space/ HTTP/1.1" 200 9435
2019-09-20 23:11:31,968 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 22185
2019-09-20 23:11:32,639 basehttp.log_message 154 INFO    => "GET /operation/share_space/ HTTP/1.1" 200 9435
2019-09-20 23:11:33,154 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 22185
2019-09-20 23:11:33,565 basehttp.log_message 154 INFO    => "GET /operation/share_space/ HTTP/1.1" 200 9435
2019-09-20 23:11:34,165 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 22185
2019-09-20 23:11:35,322 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 22185
2019-09-20 23:11:35,663 basehttp.log_message 154 INFO    => "GET /operation/share_space/ HTTP/1.1" 200 9435
2019-09-20 23:11:37,144 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 22185
2019-09-20 23:11:53,133 basehttp.log_message 154 INFO    => "GET /operation/share_space/ HTTP/1.1" 200 9435
2019-09-20 23:11:53,745 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 22185
2019-09-20 23:11:54,685 basehttp.log_message 154 INFO    => "GET /operation/share_space/ HTTP/1.1" 200 9435
2019-09-20 23:11:55,444 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 22185
2019-09-20 23:31:30,297 basehttp.log_message 154 INFO    => "GET /operation/operation_board/ HTTP/1.1" 200 22185
2019-09-20 23:31:46,259 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=2 HTTP/1.1" 200 22516
2019-09-20 23:31:46,983 basehttp.log_message 154 INFO    => "GET /operation/operation_board/?current_page=3 HTTP/1.1" 200 22372
2019-09-20 23:31:49,808 basehttp.log_message 154 INFO    => "GET /operation/view_file/?file_name=fenghuo.pdf&hashed_name=fenghuo.pdf&file_type=pdf HTTP/1.1" 200 169
2019-09-20 23:32:00,017 basehttp.log_message 154 INFO    => "GET /operation/view_file/?file_name=%E5%8F%96%E4%B8%8D%E5%88%B0NeonSAN%E7%9A%84%E6%95%B0%E6%8D%AE.png&hashed_name=%E5%8F%96%E4%B8%8D%E5%88%B0NeonSAN%E7%9A%84%E6%95%B0%E6%8D%AE.png&file_type=png HTTP/1.1" 200 221
